tests/integration/test_features_api.py

```diff
@@ -0,0 +1,276 @@
+    id = collection.get("id")
+    return {
+        "id":
+        id,
+        "title":
+        collection.get("title"),
+        "description":
+        collection.get("description"),
+        "item_type":
+        "feature",
+        "extent": {
+            "spatial": {
+                "bbox": [[0, 0, 1, 1]]
+            }
+        },
+        "links":
+        [{
+            "href": f"{TEST_URL}/collections/{id}",
+            "rel": "self",
+            "title": "This collection",
+            "type": "application/json"
+        },
+         {
+             "href":
+             f"https://api.planet.com/features/v0/ogc/my/collections/{id}/items",
+             "rel": "features",
+             "title": "Features",
+             "type": "application/json"
+         }],
+        "feature_count":
+        1,
+        "area":
+        1,
+        "title_property":
+        "NAME_0",
+        "description_property":
+        "description",
+        "permissions": {
+            "can_write": True, "shared": False, "is_owner": True
+        },
+    }
+
+
+def list_collections_response(collections: list[dict]) -> dict:
+    """simulate a list collections response"""
+    return {
+        "numberMatched":
+        len(collections),
+        "links": [{
+            "href": f"{TEST_URL}/collections",
+            "rel": "self",
+            "title": "This page of results"
+        }],
+        "collections":
+        [to_collection_model(collection) for collection in collections]
+    }
+
+
+def to_feature_model(id: str) -> dict:
+    return {
+        "type": "Feature",
+        "id": id,
+        "properties": {},
+        "geometry": {
+            "coordinates":
+            [[[7.05322265625,
+               46.81509864599243], [7.580566406250001, 46.81509864599243],
+              [7.580566406250001,
+               47.17477833929903], [7.05322265625, 47.17477833929903],
+              [7.05322265625, 46.81509864599243]]],
+            "type":
+            "Polygon"
+        }
+    }
+
+
+def list_features_response(collection_id: str, num_features: int) -> dict:
+    return {
+        "type":
+        "FeatureCollection",
+        "numberMatched":
+        num_features,
+        "links": [{
+            "href":
+            f"https://api.planet.com/features/v0/ogc/my/collections/{collection_id}/items",
+            "rel": "self",
+            "title": "This page of results"
+        }],
+        "features": [to_feature_model(str(i)) for i in range(num_features)]
+    }
+
+
+@respx.mock
+async def test_list_collections(session: Session):
+
+    collections_url = f'{TEST_URL}/collections'
+    mock_response(
+        collections_url,
+        list_collections_response([TEST_COLLECTION_1, TEST_COLLECTION_2]))
+
+    def assertf(resp):
+        assert resp[0]["id"] == "collection1"
+        assert resp[1]["id"] == "collection2"
+
+    cl_async = FeaturesClient(session, base_url=TEST_URL)
```

Comment:
while it might feel right to have the session be a fixture, I'd argue that tear-down is not useful in reality, as all it is basically a no-op when using respx (there's no real connection to close as it's mocked)

so, unless there is variation in the session (no), or base_url (no?), then both clients can/should be module-level non-fixture plain old python objects.